src/transformers/models/auto/configuration_auto.py

```diff
@@ -1007,6 +1007,8 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
         kwargs["_from_auto"] = True
         kwargs["name_or_path"] = pretrained_model_name_or_path
         trust_remote_code = kwargs.pop("trust_remote_code", None)
+        code_revision = kwargs.pop("code_revision", None)
```

Comment:
Not strictly related to the issue at hand, but this cleanup makes sure `code_revision` is not recursively passed along.